docs/src/app/components/Navigation/docs.js

```diff
@@ -200,6 +200,16 @@ export default {
             },
           ],
         },
+        {
+          url: 'troubleshooting',
+          displayName: 'Troubleshooting',
+          children: [
+            {
+              url: 'node-certificates',
+              displayName: 'Sitecore CA certificates for Node.js',
```

Comment:
Is "CA certificate" a commonly known term? Personally, I had to look this up to make sure it's referring to what I thought it was referring to, as I'm used to hearing "secure/ssl/https/self-signed certificates"

Reply:
Hmm, maybe we should drop "CA"? Just worried about making the name too broad / generic since this article is all about the "NODE_EXTRA_CA_CERTS". More than likely they'll land here from one of the cross-links or searching for the specific errors.